src/parse/atom/mod.rs

```diff
@@ -1,41 +1,38 @@
 use nom::{
-    branch::alt,
-    bytes::complete::{ is_not, tag },
-    character::complete::digit1,
-    combinator::{ map, map_res },
-    sequence::{
-        preceded,
-        delimited,
-        terminated,
-        separated_pair,
-    },
+    branch,
+    bytes,
+    character,
+    combinator,
+    sequence,
     error::{
         VerboseError,
-        VerboseErrorKind::Context,
+        VerboseErrorKind,
     },
-    IResult, Err,
+    IResult,
 };
 
-use crate::expr::{ Atom, Num };
-
-
+use crate::expr::{
+    Atom,
+    Num
+};
 
 // Public
 
 pub fn atom<'a>(i: &'a str) -> IResult<&'a str, Atom, VerboseError<&'a str>> {
-    alt((
+    use combinator::map;
+    branch::alt((
         map(num, Atom::Num),
         map(lit_char, Atom::Char),
         map(string, Atom::String),
         map(symbol, Atom::Symbol),
     ))(i)
 }
 
-
-
 // String parsing
 
 fn string<'a>(i: &'a str) -> IResult<&'a str, String, VerboseError<&'a str>> {
+    use bytes::complete::tag;
+    use sequence::{terminated, preceded};
```

Comment:
Here I am breaking with the styling I had before (as in I didn't include space next to the curly brackets).  I think this is what the Rust community expects.  Regardless, I want to format all of the this with `cargo fmt` after this PR is merged, so I am not worried about it right now.  I just wanted to be clear about the style inconsistency here, and that we are going to fix it automatically anyway ;)